### PR TITLE
Fix minor errors in book

### PR DESCRIPTION
**Pull request by HartBlanc**
Just fixing some minor errors I noticed while reading the book!

### CODE REVIEW

File: book/working_with_strings.md [hunk 1]
```diff
@@ -437,4 +437,6 @@ You can color strings with the [`ansi`](/commands/docs/ansi.md) command. For exa
 ```
 
 `ansi purple_bold` makes the text a bold purple
-`ansi reset` resets the coloring to the default. (Tip: You should always end colored strings with `ansi reset`)
+`ansi reset` resets the coloring to the default.
+
+_Tip: You should always end colored strings with `ansi reset`_
```
Comment (NotTheDr01ds) [inline book/working_with_strings.md:442]:
```suggestion
::: tip
You should always end colored strings with `ansi reset`.
:::
```

File: book/configuration.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
 Nushell uses a configuration system that loads and runs two Nushell script files at launch time:
 
 - `env.nu` is used to define environment variables or write files before `config.nu` starts. For example [`$env.NU_LIB_DIRS`](/book/modules.md#dumping-files-into-directory) controls where Nu finds imports. Third party scripts, like prompts or [mise](https://mise.jdx.dev/getting-started.html#nushell), must already be saved to disk before `config.nu` can read them.
-- `config.nu` is used to add definitions, aliases, and more to the global namespace. It can also use the environment variables and constants defined in `env.nu`. If you can't decide which file to add stuff, prefer `config.nu`.
+- `config.nu` is used to add definitions, aliases, and more to the global namespace. It can also use the environment variables and constants defined in `env.nu`. If you can't decide which file to add stuff to, prefer `config.nu`.
```
Comment (NotTheDr01ds) [inline book/configuration.md:8]: Thanks!  However, the entire config chapter is getting replaced today with a rewrite for the 0.101 release.  This text no longer exists in the new version, so unfortunately it's going to be a merge conflict once that new version goes it.

File: book/pipelines.md [hunk 3]
```diff
@@ -453,7 +453,7 @@ drwxr-xr-x@  4 pengs  admin   128 14 Nov 13:42 en
 (Actually, for this simple usage you can just use [`find`](/commands/docs/find.md))
 
 ```nu
-> ls /usr/share/nvim/runtime/ | get name | find tutor | ^ls -al $in
+> ls /usr/share/nvim/runtime/ | get name | find tutor | ansi strip | ^ls -al ...$in
```
Comment (sholderbach) [inline book/pipelines.md:456]: Thanks! Putting a pin in here that we should change that once we get rid of the `find`-inserts-ANSI pathology. https://github.com/nushell/nushell/issues/11899

File: book/configuration.md (hunk 2 above)
Comment (HartBlanc) [reply book/configuration.md]: Ah, thanks for the heads up, I've removed it now

File: book/pipelines.md (hunk 3 above)
Comment (HartBlanc) [reply book/pipelines.md:456]: I'm glad to see this is being reviewed - it certainly tripped me up!

**Comment by NotTheDr01ds** [comment]
Thanks for reading, reviewing, catching, and fixing these!  Merged!

**Pull request merged**